packages/init/src/lib/framework.ts

```diff
@@ -94,14 +94,14 @@ export const FRAMEWORKS: Record<string, Framework> = {
 			"@slicemachine/adapter-nuxt2": npmDistributionTag,
 		},
 	},
-	"nuxt-3": {
-		name: "Nuxt 3",
-		sliceMachineTelemetryID: "nuxt-3",
+	nuxt: {
+		name: "Nuxt",
+		sliceMachineTelemetryID: "nuxt",
```

Comment:
💡 #idea

I guess we would need to communicate to Côme about the change to ensure we have all dashboard updated 🧐

Reply:
Definitely! I just thought it'd make sense to update it because moving forward `nuxt` will mean Nuxt 3 and above, like `next` for Next.js ☺️